bark/world/evaluation/rss_interface.cpp

```diff
@@ -0,0 +1,481 @@
+// Copyright (c) 2019 fortiss GmbH, Julian Bernhard, Klemens Esterle, Patrick
+// Hart, Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "bark/world/evaluation/rss_interface.hpp"
+
+using ::ad::map::point::ENUCoordinate;
+using ::ad::map::route::FullRoute;
+using ::ad::physics::Acceleration;
+using ::ad::physics::Distance;
+using ::ad::physics::Duration;
+
+namespace bark {
+namespace world {
+namespace evaluation {
+
+bool RssInterface::initializeOpenDriveMap(
+    const std::string& opendrive_file_name) {
+  std::ifstream opendrive_file(opendrive_file_name);
+  std::string opendrive_file_content =
+      std::string{std::istreambuf_iterator<char>(opendrive_file),
+                  std::istreambuf_iterator<char>()};
+  ::ad::map::access::cleanup();
+
+  // the 2nd argument is the value of narrowing overlapping between two lanes,
+  // it is only relevent if the map has intersection
+  bool result = ::ad::map::access::initFromOpenDriveContent(
+      opendrive_file_content, 0.01,
+      ::ad::map::intersection::IntersectionType::TrafficLight,
```

Comment:
why traffic light?

Reply:
I set it to Unknow intersection type now.